tilequeue/format/__init__.py

```diff
@@ -102,14 +119,22 @@ def format_vtm(fp, feature_layers, zoom, bounds_merc, bounds_lnglat):
 # package of tiles as a metatile zip
 zip_format = OutputFormat('ZIP Metatile', 'zip', 'application/zip',
                           None, None, None)
+# MVT, but written out by coanacatl/wagyu
+coanacatl_format = OutputFormat('MVT/Coanacatl', 'mvt',
+                                'application/x-protobuf', format_coanacatl,
+                                5, supports_shapely_geom)
```

Comment:
Just a question: What is the "5" here?

Reply:
It's the "priority" of the driver. I'm not sure that it's used anywhere any more, but I didn't want to rip it out, just in case.